### ISSUE TITLE
const-folding

### ISSUE THREAD

**Issue by mqyhlkahu** [dflook/python-minifier#91]
A possible improvement would be const-folding expressions that would be shorter as a constant value.

Example:
```py
SOME_VARIABLE = (((123 * 456) + 789) // 10) & 0xabcdef
```
can be transformed into
```py
SOME_VARIABLE=1063
```

#
Note that the Python interpreter will do this at compile-time anyway, so it won't have any effect on the code that is actually run.
```pycon
>>> import dis
>>> dis.dis("(((123 * 456) + 789) // 10) & 0xabcdef")
```
```py
  0           0 RESUME                   0

  1           2 LOAD_CONST               0 (1063)
              4 RETURN_VALUE
```

**Comment by dflook** [comment]
Thanks for creating an issue @kotnen, keep any eye out for this in the next release 😄

**Comment by dflook** [comment]
This was included in v2.10
